### DISCUSSION TITLE
Spesa "pagata"

### DISCUSSION THREAD

**Discussion by dcesca** [fattureincloud/api#419] [Q&A] [answered]
Buongiorno,
vorrei filtrare le spese che sono da pagare/pagate ma non trovo il campo corretto da usare per la query o nemmeno il valore che esprime questo stato nell'oggetto dei documenti ricevuti.
In sostanza vorrei fare quello che fanno i tasti "tutte","da pagare" e "pagate" (es  url da interfacci fatture in cloud https://secure.fattureincloud.it/ajaxtables/expenses.php?query=&cost_center=KmFsbCo%3D&from=3&to=3&marked=-1&invoiceFamily=1&category=VHV0dGkgZ2xpIGFjcXVpc3Rp&amountType=0&orderBy=0&orderOrd=0&page=1)

Potete aiutarmi?

**Comment by emajo** [comment, marked as answer]
Ciao @dcesca, puoi trovare la risposta che cerca all'interno di [quest'altra discussion](https://github.com/fattureincloud/api/discussions/171).

Ci faccia sapere se risolve,
grazie,
Emanuele.
